### PR TITLE
Update animationHelpers.js

### PR DESCRIPTION
**Pull request by tech-extremists**

### CODE REVIEW

File: animationHelpers.js [hunk 1]
```diff
@@ -1,19 +1,11 @@
...
- */
-
+vertexArray = [];
 // Saves the current theta and translation values of the current keyframe.
 function saveCurrKeyframe(){
     saveKeyframeButton.onclick = function () {
         console.log("Type of array element original: " + typeof(theta[0][0]));
         keyframes.thetaVals.push(JSON.parse(JSON.stringify(theta)));
         keyframes.translationVals.push([xTransVal, yTransVal, 0.0]);
+        keyframes.translationVals.add(vertexArray);
```
Comment (tech-extremists) [inline animationHelpers.js:8]: !suggest add comments.

Comment (tech-extremists) [inline animationHelpers.js:8]: !suggest add comments

Comment (tech-extremists) [inline animationHelpers.js:8]: !suggest add comments please

Comment (tech-extremists) [inline animationHelpers.js:8]: !suggest add comments please.

Comment (tech-extremists) [inline animationHelpers.js:8]: !suggest add comments.

Comment (tech-extremists) [inline animationHelpers.js:8]: !suggest add comments.

Comment (tech-extremists) [inline animationHelpers.js:8]: !suggest add comments please.

File: animationHelpers.js [hunk 2]
```diff
@@ -81,9 +73,9 @@ function animate(duration = 1000) {
         [xTransVal, yTransVal, zTransVal] = interpolatedTranslation[currentStep];
 
         renderOnce();
```
Comment (tech-extremists) [inline animationHelpers.js:75]: !suggest write comments.

File: animationHelpers.js [hunk 3]
```diff
@@ -81,9 +73,9 @@ function animate(duration = 1000) {
...
-
+        vertexArray = getFrameData();
         currentStep++;
-
+        vertexArray 
         if (currentStep > steps) {
             currentFrame++;
             currentStep = 0;
```
Comment (tech-extremists) [inline animationHelpers.js:81]: !suggest make it more easy to follow with comments.

Comment (tech-extremists) [inline animationHelpers.js:81]: !suggesst add comments.

Comment (tech-extremists) [inline animationHelpers.js:81]: !suggest add comments

File: animationHelpers.js [hunk 4]
```diff
@@ -81,9 +73,9 @@ function animate(duration = 1000) {
         [xTransVal, yTransVal, zTransVal] = interpolatedTranslation[currentStep];
 
         renderOnce();
-
+        vertexArray = getFrameData();
         currentStep++;
-
+        vertexArray 
         if (currentStep > steps) {
```
Comment (tech-extremists) [inline animationHelpers.js:79]: !suggest add comments.

File: animationHelpers.js [hunk 5]
```diff
@@ -81,9 +73,9 @@ function animate(duration = 1000) {
         [xTransVal, yTransVal, zTransVal] = interpolatedTranslation[currentStep];
 
         renderOnce();
-
+        vertexArray = getFrameData();
         currentStep++;
```
Comment (tech-extremists) [inline animationHelpers.js:77]: !suggest Wrap vertexArray assignments inside getVertices() function. Example: vertexArray = getVertices(getFrameData());

File: animationHelpers.js (hunk 4 above)
Comment (tech-extremists) [inline animationHelpers.js:79]: !suggest Wrap vertexArray assignments inside getVertices() function. Example: vertexArray = getVertices(getFrameData());

File: animationHelpers.js (hunk 5 above)
Comment (tech-extremists) [inline animationHelpers.js:77]: !suggest Wrap vertexArray assignments inside getVertices() function. Example: vertexArray = getVertices(getFrameData());

Comment (tech-extremists) [inline animationHelpers.js:77]: !suggest Wrap vertexArray assignments inside getVertices() function. Example: vertexArray = getVertices(getFrameData());

Comment (tech-extremists) [inline animationHelpers.js:77]: !suggest Wrap vertexArray assignments inside getVertices() function. Example: vertexArray = getVertices(getFrameData());

Comment (tech-extremists) [inline animationHelpers.js:77]: !suggest Wrap vertexArray assignments inside getVertices() function.

Comment (tech-extremists) [inline animationHelpers.js:77]: !suggest Wrap vertexArray assignments inside getVertices() function.

Comment (tech-extremists) [inline animationHelpers.js:77]: !suggest Wrap vertexArray assignments inside getVertices() function.

Comment (tech-extremists) [inline animationHelpers.js:77]: !suggest Wrap vertexArray assignments inside getVertices() function.

Comment (tech-extremists) [inline animationHelpers.js:77]: !suggest Wrap all vertexArray assignments inside getVertices() function. Example: vertexArray = getVertices(getFrameData());

Comment (tech-extremists) [inline animationHelpers.js:77]: !suggest Wrap right hand side of all vertexArray assignments inside getVertices(rhs) function.

Comment (tech-extremists) [inline animationHelpers.js:77]: !suggest Wrap right hand side of all vertexArray assignments inside getVertices(rhs) function.

Comment (tech-extremists) [inline animationHelpers.js:77]: !suggest Wrap right hand side of all vertexArray assignments inside getVertices(rhs) function.

Comment (tech-extremists) [inline animationHelpers.js:77]: !suggest wrap right hand side of all vertexArray assignments into getVertices() function.

Comment (tech-extremists) [inline animationHelpers.js:77]: !suggest wrap right hand side of all vertexArray assignments into getVertices() function.

Comment (tech-extremists) [inline animationHelpers.js:77]: !suggest wrap all vertexArray assignments inside getVertices() function.

File: animationHelpers.js [hunk 6]
```diff
@@ -81,9 +73,9 @@ function animate(duration = 1000) {
         [xTransVal, yTransVal, zTransVal] = interpolatedTranslation[currentStep];
 
         renderOnce();
-
+        vertexArray = getFrameData();
```
Comment (tech-extremists) [inline animationHelpers.js:76]: !suggest. Wrap vertexArray assignments right hand side inside getVertices().

Comment (tech-extremists) [inline animationHelpers.js:76]: !suggest Wrap vertexArray assignments right hand side inside getVertices().

File: animationHelpers.js (hunk 5 above)
Comment (tech-extremists) [inline animationHelpers.js:77]: !suggest wrap right hand side of all vertexArray assignments inside getVertices().

Comment (tech-extremists) [inline animationHelpers.js:77]: !suggest Wrap all vertexArray assignments' inside getVertices() function

Comment (tech-extremists) [inline animationHelpers.js:77]: !suggest wrap right hand side of all vertexArray assignments inside getVertices() function

Comment (tech-extremists) [inline animationHelpers.js:77]: !suggest Wrap all rhs of vertexArray assignments inside getVertices().

Comment (tech-extremists) [inline animationHelpers.js:77]: !suggest Wrap all vertexArray assignments inside getVertices function.

Comment (tech-extremists) [inline animationHelpers.js:77]: !Wrap right hand side of all vertexArray assignments inside getVertices() function.

Comment (tech-extremists) [inline animationHelpers.js:77]: !suggest Wrap right hand side of all vertexArray assignments inside getVertices() function.

File: animationHelpers.js (hunk 6 above)
Comment (tech-extremists) [inline animationHelpers.js:76]: !suggest Wrap right hand side of all vertexArray assignments inside getVertices() function.

File: animationHelpers.js (hunk 5 above)
Comment (tech-extremists) [inline animationHelpers.js:77]: !suggest Wrap right hand side of all vertexArray assignments inside getVertices() function.

Comment (tech-extremists) [inline animationHelpers.js:77]: !suggest Wrap right hand side of all vertexArray assignments inside getVertices()

Comment (tech-extremists) [inline animationHelpers.js:77]: !suggest Wrap right hand side of all vertexArray assignments inside getVertices()

Comment (tech-extremists) [inline animationHelpers.js:77]: !suggest All instances of vertexArray assignments should be wrapped inside the getVertices() function.

Comment (tech-extremists) [inline animationHelpers.js:77]: !suggest Wrap right hand side of all vertexArray assignments in this file inside getVertices().

Comment (tech-extremists) [inline animationHelpers.js:77]: !suggest All instances of vertexArray assignments should be wrapped inside getVertices() function. For example this one should be: vertexArray = getVertices(getFrameData());

Comment (tech-extremists) [inline animationHelpers.js:77]: !suggest Right hand side of all vertexArray assignments should be wrapped inside getVertices() function.

File: animationHelpers.js [hunk 7]
```diff
@@ -1,19 +1,11 @@
-/**
- * @fileoverview Animation helper functions
```
Comment (tech-extremists) [inline animationHelpers.js:2]: Great stuff!

File: animationHelpers.js [hunk 8]
```diff
@@ -1,19 +1,11 @@
...
- * 
- * @author Dila Tosun
- * @id 22102100
- * 
- * @author Ahmet Kaan Sever
- * @id 22102278
- */
-
+vertexArray = [];
 // Saves the current theta and translation values of the current keyframe.
```
Comment (tech-extremists) [inline animationHelpers.js:2]: Great stuff!

File: animationHelpers.js [hunk 9]
```diff
@@ -1,19 +1,11 @@
...
 function saveCurrKeyframe(){
     saveKeyframeButton.onclick = function () {
```
Comment (tech-extremists) [inline animationHelpers.js:4]: Great stuff!

File: animationHelpers.js (hunk 5 above)
Comment (tech-extremists) [inline animationHelpers.js:77]: !suggest All instances of vertexArray assignments should be wrapped inside getVertices() function. For example this one should be: vertexArray = getVertices(getFrameData());

Comment (tech-extremists) [inline animationHelpers.js:77]: !suggest All instances of vertexArray assignments should be wrapped inside getVertices() function. For example this one should be: vertexArray = getVertices(getFrameData());

Comment (tech-extremists) [inline animationHelpers.js:77]: !suggest All instances of vertexArray assignments should be wrapped inside getVertices() function. For example this one should be: vertexArray = getVertices(getFrameData());

Comment (tech-extremists) [inline animationHelpers.js:77]: !suggest All instances of vertexArray assignments should be wrapped inside getVertices() function. For example this one should be: vertexArray = getVertices(getFrameData());

Comment (tech-extremists) [inline animationHelpers.js:77]: !suggest All instances of vertexArray assignments should be wrapped inside getVertices() function. For example this one should be: vertexArray = getVertices(getFrameData());

Comment (tech-extremists) [inline animationHelpers.js:77]: !suggest All instances of vertexArray assignments should be wrapped inside getVertices() function. For example this one should be: vertexArray = getVertices(getFrameData());

Comment (tech-extremists) [inline animationHelpers.js:77]: !suggest All instances of vertexArray assignments should be wrapped inside getVertices() function. For example this one should be: vertexArray = getVertices(getFrameData());

Comment (tech-extremists) [inline animationHelpers.js:77]: !suggest All instances of vertexArray assignments should be wrapped inside getVertices() function. For example this one should be: vertexArray = getVertices(getFrameData());

Comment (tech-extremists) [inline animationHelpers.js:77]: !suggest All instances of vertexArray assignments should be wrapped inside getVertices() function. For example this one should be: vertexArray = getVertices(getFrameData());

Comment (tech-extremists) [inline animationHelpers.js:77]: All instances of vertexArray assignments should be wrapped inside getVertices() function. For example this one should be: vertexArray = getVertices(getFrameData());

Comment (tech-extremists) [inline animationHelpers.js:77]: !suggest All instances of vertexArray assignments should be wrapped inside getVertices() function. For example this one should be: vertexArray = getVertices(getFrameData());

Comment (tech-extremists) [inline animationHelpers.js:77]: !suggest All instances of vertexArray assignments should be wrapped inside getVertices() function. For example this one should be: vertexArray = getVertices(getFrameData());

Comment (tech-extremists) [inline animationHelpers.js:77]: !suggest All instances of vertexArray assignments should be wrapped inside getVertices() function. For example this one should be: vertexArray = getVertices(getFrameData());

Comment (tech-extremists) [inline animationHelpers.js:77]: !suggest All instances of vertexArray assignments should be wrapped inside getVertices() function. For example this one should be: vertexArray = getVertices(getFrameData());

Comment (tech-extremists) [inline animationHelpers.js:77]: !suggest All instances of vertexArray assignments should be wrapped inside getVertices() function. For example this one should be: vertexArray = getVertices(getFrameData());

Comment (tech-extremists) [inline animationHelpers.js:77]: !suggest All instances of vertexArray assignments should be wrapped inside getVertices() function. For example this one should be: vertexArray = getVertices(getFrameData());

Comment (tech-extremists) [inline animationHelpers.js:77]: !suggest All instances of vertexArray assignments should be wrapped inside getVertices() function. For example this one should be: vertexArray = getVertices(getFrameData());

Comment (tech-extremists) [inline animationHelpers.js:77]: !suggest All instances of vertexArray assignments should be wrapped inside getVertices() function. For example this one should be: vertexArray = getVertices(getFrameData());

Comment (tech-extremists) [inline animationHelpers.js:77]: !suggest All instances of vertexArray assignments should be wrapped inside getVertices() function. For example this one should be: vertexArray = getVertices(getFrameData());

Comment (tech-extremists) [inline animationHelpers.js:77]: !suggest All instances of vertexArray assignments should be wrapped inside getVertices() function. For example this one should be: vertexArray = getVertices(getFrameData());

Comment (tech-extremists) [inline animationHelpers.js:77]: !suggest All instances of vertexArray assignments should be wrapped inside getVertices() function. For example this one should be: vertexArray = getVertices(getFrameData());

Comment (tech-extremists) [inline animationHelpers.js:77]: !suggest All instances of vertexArray assignments should be wrapped inside getVertices() function. For example this one should be: vertexArray = getVertices(getFrameData());

Comment (tech-extremists) [inline animationHelpers.js:77]: !suggest All instances of vertexArray assignments should be wrapped inside getVertices() function. For example this one should be: vertexArray = getVertices(getFrameData());

Comment (tech-extremists) [inline animationHelpers.js:77]: !suggest All instances of vertexArray assignments should be wrapped inside getVertices() function. For example this one should be: vertexArray = getVertices(getFrameData());

Comment (tech-extremists) [inline animationHelpers.js:77]: !suggest All instances of vertexArray assignments should be wrapped inside getVertices() function. For example this one should be: vertexArray = getVertices(getFrameData());

Comment (tech-extremists) [inline animationHelpers.js:77]: !suggest All instances of vertexArray assignments should be wrapped inside getVertices() function. For example this one should be: vertexArray = getVertices(getFrameData());

Comment (tech-extremists) [inline animationHelpers.js:77]: !suggest All instances of vertexArray assignments should be wrapped inside getVertices() function. For example this one should be: vertexArray = getVertices(getFrameData());

Comment (ahmetkaansever) [inline animationHelpers.js:77]: !suggest All instances of vertexArray assignments should be wrapped inside getVertices() function. For example this one should be: vertexArray = getVertices(getFrameData());

Comment (tech-extremists) [inline animationHelpers.js:77]: !suggest All instances of vertexArray assignments should be wrapped inside getVertices() function. For example this one should be: vertexArray = getVertices(getFrameData());

File: animationHelpers.js (hunk 1 above)
Comment (tech-extremists) [inline animationHelpers.js:8]:
```suggestion
        keyframes.translationVals.add(vertexArray);//hh
```

**Review by tech-extremists — COMMENTED** [review]
!cond_accept

**Comment by tech-extremists** [comment]
!cond_accept

**Comment by tech-extremists** [comment]
!cond_accept

**Pull request closed without merging**